ckanext/validation/helpers.py

```diff
@@ -67,6 +72,19 @@ def get_validation_badge(resource, in_listing=False):
         title=resource.get('validation_timestamp', ''))
 
 
+def _get_schema_or_default_schema(resource):
+
+    if asbool(resource.get('align_default_schema')):
+        schema = get_default_schema(resource['package_id'])
```

Comment:
Is default schema, part of this plugin or another? I'm losing track on the splits on this work.

Reply:
It's in this one. You may be thinking of the validation schema generator plugin, that's something different.